pkg/controller/livenessprobemapnodeprobe/livenessprobemapnodeprobe.go

```diff
@@ -0,0 +1,139 @@
+package livenessprobemapnodeprobe
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+
+	"github.com/openkruise/kruise/pkg/util"
+	utilclient "github.com/openkruise/kruise/pkg/util/client"
+)
+
+const (
+	concurrentReconciles = 10
+	controllerName       = "livenessprobemapnodeprobe-controller"
```

Comment:
rename controller to more simplified form LivenessNodeProbeController , also plz rename the package and file name

Reply:
done